dtest_config.py

```diff
@@ -102,6 +100,17 @@ def setup(self, config):
 
         self.latest_config = False if self.configuration_yaml is None else self.configuration_yaml.endswith("cassandra_latest.yaml")
 
+        self.apply_to_env(os.environ, "JVM_EXTRA_OPTS")
+
+    def apply_to_env(self, env, key="JVM_OPTS"):
+        current = env.get(key) or ""
+        if self.sstable_format:
+            default_sstable_format_prop = " -Dcassandra.sstable.format.default=" + self.sstable_format
+            if not current.__contains__("-Dcassandra.sstable.format.default"):
+                env.update({key: (env.get(key) or "") + default_sstable_format_prop})
+            else:
+                logger.debug("Skipped adding {} because it is already in the env key {}: {}".format(default_sstable_format_prop, key, current))
+
```

Comment:
in https://github.com/datastax/cassandra-dtest/pull/78/commits/0f0035efd5f79c5e827aba8b7016a80534f2821f

@jacek-lewandowski , out of curiosity…  why don't we have this upstream ?  (given upstream now has both `big` and `bti` now)

Reply:
@djatnieks ,  can we squash (during this rebase) this commit back into https://github.com/datastax/cassandra-dtest/pull/78/commits/b3b67b935df6b2ada8e9d676afbd65c4132ebef7 ?

Reply:
@michaelsembwever Sorry, squash which commit into https://github.com/datastax/cassandra-dtest/commit/b3b67b935df6b2ada8e9d676afbd65c4132ebef7? The 0f0035e commit?

Probably, but I think @jacek-lewandowski could answer as he authored both, iiuc?